src/test/suite/intraCellSetSelection.test.ts

```diff
@@ -1,18 +1,12 @@
 import * as assert from "assert";
 import * as vscode from "vscode";
-import * as sinon from "sinon";
 import { getCursorlessApi } from "../../util/getExtensionApi";
 import { openNewNotebookEditor } from "../openNewEditor";
-import sleep from "../../util/sleep";
+import { sleepWithBackoff, standardSuiteSetup } from "./standardSuiteSetup";
 
 // Check that setSelection is able to focus the correct cell
 suite("Within cell set selection", async function () {
-  this.timeout("100s");
-  this.retries(3);
```

Comment:
In particular, notice that the test that was failing in CI was only retrying 3 times, instead of the usual 5